src/main/controllers/SjpPressListController.ts

```diff
@@ -40,11 +40,23 @@ export default class SjpPressListController {
 
             const pageLanguage = publicationService.languageToLoadPageIn(metaData.language, req.lng);
             const showDownloadButton = await listDownloadService.generateFiles(artefactId, req.user);
+            const url = publicationService.getListTypes().get(metaData.listType).url;
+
+            let languageResource = {
+                ...req.i18n.getDataByLanguage(pageLanguage)['single-justice-procedure-press'],
+                ...req.i18n.getDataByLanguage(pageLanguage)['sjp-common'],
+                ...req.i18n.getDataByLanguage(pageLanguage)['list-template'],
+            };
+
+            if (url === 'sjp-press-list-new-cases') {
```

Comment:
Feel free to ignore, however I'm thinking rather than URL the list type comparison here would be better. (The URL comparison took me a few seconds to work out, where as list type would be clear)

Reply:
Agreed it is better to use list type. This has been updated